packages/functional-tests/tests/oauth/webchannel.spec.ts

```diff
@@ -56,15 +63,22 @@ test.describe('severity-1 #smoke', () => {
       testAccountTracker,
     }) => {
       const credentials = await testAccountTracker.signUp();
-      const customEventDetail = createCustomEventDetail(
-        FirefoxCommand.FxAStatus,
-        {
-          capabilities: {
-            engines: ['bookmarks', 'history'],
+      const customEventDetail: FxAStatusResponse = {
+        id: 'account_updates',
+        message: {
+          command: FirefoxCommand.FxAStatus,
+          data: {
+            clientId: '',
+            capabilities: {
+              choose_what_to_sync: true,
+              multiService: true,
```

Comment:
Ah that's interesting. I checked my notes again, I had pulled that info [from this in ecosystem-platform](https://mozilla.github.io/ecosystem-platform/reference/webchannels#capabilities) and [here's a related ticket](https://mozilla-hub.atlassian.net/browse/FXA-8641). (There was a Slack conversation around sync optional too recently but `multiService` wasn't explicitly mentioned). Maybe those docs are just out of date, because yeah as noted in that Jira ticket too we're not accounting for that now. So - fine with me to leave it how you have it.